XFCorePostProcessor.Tasks/XFCorePostProcessor.Tasks.csproj

```diff
@@ -5,6 +5,14 @@
 		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <WarningsAsErrors />
```

Comment:
what's the difference with TreatWarningsAsErrors ?

Reply:
From https://github.com/NuGet/Home/issues/6288

> WarningsAsErrors accepts comma or semi-colon separated NU codes to convert individual codes into errors. e.g. - <WarningsAsErrors>NU16095,NU1701</WarningsAsErrors>

If that's true then this would be specifying an empty set of warnings to treat as errors and so not the droids we're looking for. So.. prob wanna add `TreatWarningsAsErrors` and drop this..